OpenOversight/tests/test_email_client.py

```diff
@@ -0,0 +1,50 @@
+from unittest.mock import MagicMock
```

Comment:
There are some other email tests, I'd like to have those in here as well.

Reply:
Do you mean existing email client tests in other files or tests for the EmailProvider subclasses? I'm adding the additional tests for the Gmail and SMTP email providers

Reply:
Really, just this test: `test_email_create_message`.

Reply:
Moved this test to test_email_client.py!